lib/vsc/install/shared_setup.py

```diff
@@ -144,13 +144,19 @@ def _log(self, level, msg, args):
 
 RELOAD_VSC_MODS = False
 
-VERSION = '0.9.10'
+VERSION = '0.9.11'
 
 log.info('This is (based on) vsc.install.shared_setup %s' % VERSION)
 
 # list of non-vsc packages that do not need python- prefix for correct rpm dependencies
 # vsc packages should be handled with clusterbuildrpm
-NO_PREFIX_PYTHON_BDIST_RPM = []
+# dependencies starting with python- are also not re-prefixed
+NO_PREFIX_PYTHON_BDIST_RPM = ['pbs_python']
+
+# Hardcode map of python dependency prefix to their rpm python- flavour prefix
+PYTHON_BDIST_RPM_PREFIX_MAP = {
+    'pycrypto': 'python-crypto',
```

Comment:
Maybe we should have a little intro on this, so people know they also have to change things here?